src/main/java/com/classwork/csj/controller/CartController.java

```diff
@@ -0,0 +1,62 @@
+package com.classwork.csj.controller;
+
+import com.classwork.csj.entity.Cart;
+import com.classwork.csj.entity.CartItem;
+import com.classwork.csj.entity.Product;
+import com.classwork.csj.exceptions.ItemNotFoundException;
+import com.classwork.csj.service.CartService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("cart")
+public class CartController {
+
+    @Autowired
+    CartService cartService;
+
+    @GetMapping("list")
+    public ResponseEntity<Cart> getCart(@RequestHeader("user-id") int userId) {
```

Comment:
Just curious tho... Why header?

Reply:
That's the id of the user that is logged in. It should come from a token of some sort but that hasn't been implemented yet so I just used that as a makeshift.